guides/03-Debugging/README.md

```diff
@@ -38,7 +38,7 @@ Use a disciplined approach - ask yourself if you've made an assumption, and test
 
 ## 2.1 Understand and get it running
 
-Open "BuggyProject" up in a text editor
+Open "[BuggyProject](https://github.com/madetech/learn/tree/master/guides/03-Debugging/BuggyProject)" up in a text editor
```

Comment:
I like relative paths, but otherwise looks good to me.

```suggestion
Open "[BuggyProject](BuggyProject)" up in a text editor
```

Reply:
Had a look and this does not work on the live site: https://learn.madetech.com/guides/03-Debugging/BuggyProject/